versions/draft-3/SPEC.md

```diff
@@ -1018,12 +1018,34 @@ task memory_test {
 
 ```
 $parameter_meta = 'parameter_meta' $ws* '{' ($ws* $parameter_meta_kv $ws*)* '}'
-$parameter_meta_kv = $identifier $ws* '=' $ws* $string
+$parameter_meta_kv = $identifier $ws* ':' $ws* $meta_value
+$meta_value = $string | $number | $boolean | 'null' | $meta_object | $meta_array
+$meta_object = '{}' | '{' $meta_kv_pair (, $meta_kv_pair)* '}'
+$meta_kv_pair = $string ':' $meta_value
```

Comment:
Any reason for not reusing `parameter_meta_kv` instead of `meta_kv_pair` here? 

Whether `parameter_meta_kv` has a `$string` literal or an `$identifier` I wouldn't really mind, I'd just prefer consistency.